hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/node/SCMNodeManager.java

```diff
@@ -121,9 +121,10 @@ public class SCMNodeManager implements NodeManager {
   private final SCMStorageConfig scmStorageConfig;
   private final NetworkTopology clusterMap;
   private final DNSToSwitchMapping dnsToSwitchMapping;
-  private final boolean useHostname;
   private final ConcurrentHashMap<String, Set<String>> dnsToUuidMap =
       new ConcurrentHashMap<>();
+  private final ConcurrentHashMap<String, Set<String>> hostNmToUuidMap =
```

Comment:
Prefer interface for variable type and please avoid obscure abbreviation that only saves 2 characters:

```suggestion
  private final Map<String, Set<String>> hostNameToUuidMap =
```